libraries/AP_Scripting/examples/qnh_alt.lua

```diff
@@ -0,0 +1,51 @@
+--[[
+    example that sends QNH altitude to GCS as a NAMED_VALUE_FLOAT QNG_ALT_FT in feet
+
+    Note: operator must be QNH_PRESSURE in hPa before each flight!
+--]]
+
+local MAV_SEVERITY = {EMERGENCY=0, ALERT=1, CRITICAL=2, ERROR=3, WARNING=4, NOTICE=5, INFO=6, DEBUG=7}
+
+local PARAM_TABLE_KEY = 93
+local PARAM_TABLE_PREFIX = "QNH_"
+
+-- add a parameter and bind it to a variable
+function bind_add_param(name, idx, default_value)
+   assert(param:add_param(PARAM_TABLE_KEY, idx, name, default_value), string.format('could not add param %s', name))
+   return Parameter(PARAM_TABLE_PREFIX .. name)
+end
+
+-- setup script specific parameters
+assert(param:add_table(PARAM_TABLE_KEY, PARAM_TABLE_PREFIX, 8), 'could not add param table')
+
+--[[
+  // @Param: QNH_PRESSURE
+  // @DisplayName: QNH pressure in hPa
+  // @Description: QNH pressure in hPa
```

Comment:
at the home/takeoff location?

Reply:
QNH is defined by aviation standards, and provided by the aviation regulator

Reply:
I was suggesting that you add that to the description.